recipes/server-thread/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "server-thread" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1b648b99f8523f7477010e4df54086603306e2709e77de9533b52c40e9eaa233
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

See: https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip, or use grayskull to generate the recipe.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```